docs/release-notes/v12.2.2.md

```diff
@@ -0,0 +1,146 @@
+---
+id: v12.2.2
+title: Version 12.2.2 Release Notes
+sidebar_label: v12.2.2
+image: "img/release-note-link-preview.png"
+---
+
+_February 29th, 2024_
+
+## Console
+
+### New User events for the Webhook section
+
+New user management events are now available in webhooks section. This will allow a Company Owner to configure and trigger specific webhooks when changes to user access management occur.
```

Comment:
```suggestion
New user management events are now available in Webhooks section. This will allow a Company Owner to configure and trigger specific webhooks when changes to user access management occur.
```